app-drawer-layout/app-drawer-layout.html

```diff
@@ -151,7 +151,11 @@
           value: '640px'
         },
 
-        _narrow: Boolean
+        narrow: {
```

Comment:
Add doc, e.g. `Returns true if it is in narrow layout. This is useful if you need to show/hide elements based on the layout.`